libs/infra-next-server/src/lib/proxy.ts

```diff
@@ -1,17 +1,13 @@
 import { Express } from 'express'
-import type { Config } from 'http-proxy-middleware'
-
+import { createProxyMiddleware } from 'http-proxy-middleware'
 export const setupProxy = async (
   app: Express,
-  proxyConfig: { [context: string]: Config } | undefined,
+  proxyConfig: { [context: string]: any } | undefined,
   dev: boolean,
 ) => {
   if (!proxyConfig || !dev) {
     return
   }
-  const { default: createProxyMiddleware } = await import(
-    'http-proxy-middleware'
-  )
```

Comment:
The idea here was to code-split this in production builds to not evaluate unused code on boot.